src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -276,6 +276,30 @@ void SetGlobalPropertiesRequest::on_event(const event_engine::Event& event) {
   }
 }
 
+bool SetGlobalPropertiesRequest::PrepareResultForMobileResponse(
+    ResponseInfo& ui_info, ResponseInfo& tts_info) const {
+  const bool result =
+      CommandRequestImpl::PrepareResultForMobileResponse(ui_info, tts_info);
+  if (ui_info.is_ok && tts_info.is_unsupported_resource) {
+    return true;
+  }
+  return result;
+}
+
+mobile_apis::Result::eType
+SetGlobalPropertiesRequest::PrepareResultCodeForResponse(
+    const ResponseInfo& ui_info, const ResponseInfo& tts_info) {
+  if (ui_info.is_ok && tts_info.is_unsupported_resource) {
+    return mobile_apis::Result::WARNINGS;
+  }
+  if (HmiInterfaces::STATE_AVAILABLE == tts_info.interface_state &&
+      tts_info.is_unsupported_resource) {
+    tts_response_info_ = "Unsupported phoneme type sent in a prompt";
```

Comment:
@AKalinich-Luxoft Again I do not see any mention of this. Please give me an exact requirement specifying this. Otherwise I do not agree with you. Also I think these requirements you are citing are not for the correct project.